taxi/tests/test_admin.py

```diff
@@ -0,0 +1,28 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase, Client
+from django.urls import reverse
+
+
+class AdminSiteTests(TestCase):
+    def setUp(self) -> None:
+        self.client = Client()
+        self.admin_user = get_user_model().objects.create_superuser(
+            username="admin",
+            password="admin123",
+        )
```

Comment:
When creating a superuser with `create_superuser`, it's common for Django's user model to require an email field. Ensure that the email field is included if it's required by your user model.